test/helpers/spectron/index.ts

```diff
@@ -47,6 +47,25 @@ export async function closeWindow(t: any) {
   await t.context.app.browserWindow.close();
 }
 
+// Try to click element multiple times
+// A gross solution for the situation when the loading element overlaps the target element
+export async function clickWhenReady(t: any, selector: string) {
+  await t.context.app.client.waitForVisible(selector);
+  let attempts = 5;
+  let err: Error;
+  while (attempts--) {
+    try {
+      await t.context.app.client.click(selector);
+      return;
+    } catch (e) {
+      err = e;
+      await sleep(500);
+    }
+  }
+  console.error(`unable to click ${selector}`);
+  throw err;
+}
+
```

Comment:
i'll be honest i really don't like this solution

Reply:
Replaced this